src/models/captured_graph_pool.cpp

```diff
@@ -1,4 +1,3 @@
-#include "onnxruntime_c_api.h"
```

Comment:
Cleaned up so the only place we include the ORT API header is in GenAI's onnxruntime_api.h. Any GenAI source that needs access to ORT types should include that file.